SalaryService.md

```diff
@@ -30,6 +32,7 @@ Requests consist of:
 |`postal_code` | Optional | string | A postal code.|
 |`salary_period` | Yes | string | Possible values are "Year", "Hour". | 
 |`country` | Yes | string |An ISO-3166 country code.|
+|`hostiste` | Yes | string |A valid CareerBuilder hostsite.|
```

Comment:
typo in 'hostiste'

Question: is hostsite parsed? If not then what happens when you request one that doesn't exist. If so then can we enumerate the valid values here?

Reply:
Fixed